CHANGELOG/CHANGELOG-0.2.md

```diff
@@ -0,0 +1,26 @@
+# Changelog 0.2
+
+## 0.2.0 - 2022-08-17
+
+### Added
+
+- E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])
```

Comment:
As suggested by maintainers, we usually don't include tests in the changelog as they are not useful for users.

Reply:
Or at least move to "Changed", as it is indeed not about new features.